### PR TITLE
Unconstrained engine layout

### PR DESCRIPTION
**Pull request by asmega**
- This allows engines to be not width constrained and use the full width of the page.
- Rio will use this

### CODE REVIEW

File: app/views/layouts/engine.html.erb [hunk 1]
```diff
@@ -12,11 +12,7 @@
 <% end %>
 
 <%= render layout: 'layouts/unconstrained' do %>
-  <div class="l-constrained">
```
Comment (benbarnett) [inline app/views/layouts/engine.html.erb]: Is this intentional? I think this might affect a few other tools

**Comment by benlovell** [comment]
:+1:

File: app/views/layouts/engine.html.erb (hunk 1 above)
Comment (benbarnett) [inline app/views/layouts/engine.html.erb]: For example https://www.moneyadviceservice.org.uk/en/tools/mortgage-calculator relies on `l-container-tool` being there

Comment (asmega) [inline app/views/layouts/engine.html.erb]: whoops may have checked in the file unintentionally...

Comment (asmega) [inline app/views/layouts/engine.html.erb]: accidental check in has now been removed

File: app/controllers/rio_controller.rb [hunk 2]
```diff
@@ -1,4 +1,13 @@
 class RioController < EmbeddedToolsController
+  def parent_template
+    if syndicated_tool_request?
+      response.headers['X-Frame-Options'] = 'ALLOWALL'
+      'layouts/engine_syndicated'
+    else
+      'layouts/engine_unconstrained'
+    end
+  end
+
```
Comment (asmega) [inline app/controllers/rio_controller.rb]: I'm not sure it would have much benefit in this context.

**Pull request merged**